onchaind/onchain.c

```diff
@@ -749,14 +748,44 @@ static void output_spent(struct tracked_output ***outs,
 	unwatch_tx(tx);
 }
 
+static void update_resolution_depth(struct tracked_output *out, u32 depth)
+{
+	status_trace("%s/%s->%s depth %u",
+		     tx_type_name(out->tx_type),
+		     output_type_name(out->output_type),
+		     tx_type_name(out->resolved->tx_type),
+		     depth);
+
+	/* BOLT #5:
+	 *
+	 * If the HTLC output has *timed out* and not been *resolved*,
+	 * the node MUST *resolve* the output and MUST fail the
+	 * corresponding incoming HTLC (if any) once the resolving
+	 * transaction has reached reasonable depth. */
+	if (out->resolved->tx_type == OUR_HTLC_TIMEOUT_TX
```

Comment:
Can be unnested

Reply:
Had to look at the src to figure out what you were saying.  Yep, fixed, though I'm not sure it's actually much clearer.